pkg/trace/client.go

```diff
@@ -146,10 +147,27 @@ func (c *Client) WritePoint(table string, fields map[string]interface{}) {
 	writeAPI.WritePoint(p)
 }
 
-func sliceToMap(tables []string) map[string]struct{} {
+func stringToMap(tables string) map[string]struct{} {
+	parsedTables := splitAndTrimEmpty(tables, ",", " ")
 	m := make(map[string]struct{})
-	for _, s := range tables {
+	for _, s := range parsedTables {
 		m[s] = struct{}{}
 	}
 	return m
 }
+
+func splitAndTrimEmpty(s, sep, cutset string) []string {
```

Comment:
A func doc would be useful.

Reply:
good thinking, fixed in fc94cab99f5afdc24eed9f04e06621b91bfb6754